descriptors/ModuleDescriptor-template.json

```
@@ -220,6 +220,19 @@
             "circulation-storage.loans.item.delete"
           ]
         },
+        {
+          "methods": [
+            "PUT"
+          ],
+          "pathPattern": "/circulation/loans/{id}/declare-lost",
+          "permissionsRequired": [
+            "circulation.loans.item.declare.lost"
```

The established convention usually uses hyphens for the hyphentated paths and includes the HTTP method at the end of the permission.

```suggestion
            "circulation.loans.item.declare-lost.put"
```

---

done.